cypress/locators/flash/index.js

```diff
@@ -0,0 +1,10 @@
+import { FLASH_PREVIEW, MESSAGE_INPUT, TIMEOUT_INPUT } from './locators';
+
+// knobs locators
+export const messageInput = () => cy.get(MESSAGE_INPUT);
+export const timeoutInput = () => cy.get(TIMEOUT_INPUT);
+
+// component preview locators
+export const flashPreview = () => cy.iFrame(FLASH_PREVIEW);
+export const getIconPreview = icon => cy.iFrame(`[data-element="${icon}"]`);
+export const messagePreview = () => cy.iFrame('[data-element="message"]');
```

Comment:
why U don't create a new const for a those data-elements in locators ?

Reply:
fixed